lib/screens/images_to_pdf.dart

```diff
@@ -32,46 +36,71 @@ class _ImageToPdfState extends State<ImageToPdf> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      floatingActionButton: Row(
-        mainAxisAlignment: MainAxisAlignment.end,
-        children: [
-          if (_images.isNotEmpty)
-            Padding(
-              padding: const EdgeInsets.symmetric(horizontal: 4),
-              child: FloatingActionButton.extended(
-                heroTag: 'export',
-                onPressed: () => exportPdf(_images),
-                icon: const Icon(Icons.upload_file),
-                label: const Text('Export'),
-                backgroundColor: Colors.green,
-              ),
+        floatingActionButton: Padding(
+          padding: const EdgeInsets.all(16.0),
+          child: FloatingActionButton.extended(
+            onPressed: () {
+              (_images.isNotEmpty)
+                  ? exportPdf(_images)
+                  : showDialog(
+                      context: context,
+                      builder: (BuildContext context) {
+                        return SimpleDialog(
+                          title: const Text("Please select some images first."),
+                          children: [
+                            SimpleDialogOption(
+                              onPressed: () {
+                                Navigator.of(context).pop();
+                              },
+                              child: const Text("OK"),
+                            )
+                          ],
+                        );
+                      });
+            },
+            backgroundColor: primaryColor,
+            splashColor: secondaryColor,
+            elevation: 8.0,
+            icon: const Icon(Icons.upload_file),
+            label: const Text(
+              'Export',
+              style: TextStyle(
+                  color: Colors.white,
+                  fontSize: 12,
+                  fontWeight: FontWeight.w700),
             ),
-          FloatingActionButton.extended(
-            heroTag: 'addimg',
-            onPressed: getImageFromGallery,
-            icon: const Icon(Icons.add),
-            label: const Text('Add image'),
           ),
-        ],
-      ),
-      appBar: AppBar(
-        title: const Text('Images to Pdf'),
-      ),
-      body: Column(
-        children: [
-          CheckboxListTile(
-              title: const Text('Crop Image'),
-              value: _cropImage,
-              onChanged: (crop) {
-                setState(() {
-                  _cropImage = crop;
-                });
-              }),
-          Expanded(child: _buildImageList(context)),
-          //TODO:make ui better
-        ],
-      ),
-    );
+        ),
+        appBar: AppBar(
+          title: const Text(
+            'Choose Image',
+            style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
+          ),
+          actions: [
```

Comment:
remove this

Reply:
Couldn't understand what to remove? the text in the appbar?

Reply:
remove the action because feature is already in floating action button in latest one